_includes/templates/compatibility-page.md

```diff
@@ -1,3 +1,5 @@
+![{{tool.name|escape_once}}]({{tool.logo}}){:.third-party-logo}
```

Comment:
Should the description be italicized and centered?

Reply:
I like it better the current way, but I've no strong opinion here.